src/components/helper-line/helper-line.scss

```diff
@@ -20,6 +19,9 @@
     line-height: normal;
 
     color: rgb(var(--contrast-1200));
+    background-color: rgb(var(--contrast-100), 0.4);
+    backdrop-filter: blur(0.5rem);
+    -webkit-backdrop-filter: blur(0.5rem);
```

Comment:
these will make sure that the content of the helper line are readable 

https://github.com/Lundalogik/lime-elements/assets/35954987/42eb312f-a877-4bbf-aea8-8914849769b7